proc/groupby.go

```diff
@@ -238,12 +264,13 @@ func (g *GroupByAggregator) Consume(r *zson.Record) error {
 		table = make(map[string]*GroupByRow)
 		g.tables[ts] = table
 	}
+	//XXX use unsafe here to avoid sending all the string keys to GC
 	row, ok := table[string(key)]
```

Comment:
The compiler is smart enough not to generate any garbage when it sees this, so there's no need for unsafe.

Reply:
+1